frontend/src/app/choreography.service.ts

```diff
@@ -0,0 +1,132 @@
+        'Darya',
+        'Arina',
+        'Ansku',
+        'Kimi',
+        'Kille',
+        'Kimara',
+        'Katri',
+        'Lasso',
+        'Anna',
+        'Vilho',
+        'Väinö',
+        'Jaana',
+        'Ville',
+        'Niina',
+        'Marjo',
+        'Napoleon',
+        'Maria',
+        'Kari',
+        'Sari',
+        'Antti',
+        'Janne',
+        'Teppo',
+        'Matti',
+        'Joonas',
+      ],
+      frames: [{
+        subframes: JSON.parse(JSON.stringify(TEST_FRAMES)),
+        notes: ''
+      }, { subframes: JSON.parse(JSON.stringify(TEST_FRAMES)), notes: '' },
+        {
+          subframes: JSON.parse(JSON.stringify(TEST_FRAMES)),
+          notes: ''
+        }, { subframes: JSON.parse(JSON.stringify(TEST_FRAMES)), notes: '' }, {
+          subframes: JSON.parse(JSON.stringify(TEST_FRAMES)),
+          notes: ''
+        }, { subframes: JSON.parse(JSON.stringify(TEST_FRAMES)), notes: '' },
+        {
+          subframes: JSON.parse(JSON.stringify(TEST_FRAMES)),
+          notes: ''
+        }, { subframes: JSON.parse(JSON.stringify(TEST_FRAMES)), notes: '' }],
+      carpet: {
+        color: '#5151b8',
+        height: 12,
+        width: 12,
+        horizontalSegments: 12,
+        verticalSegments: 6
+      }
+    },
+    {
+      id: 2,
+      name: 'EM-karsinnat',
+      team: 'Flames',
+      people: [
+        'Roope',
+        'Olli',
+        'Darya',
+        'Arina',
+        'Ansku',
+        'Kimi',
+        'Kille',
+        'Kimara',
+        'Katri',
+        'Lasso',
+        'Anna',
+        'Vilho',
+        'Väinö',
+        'Jaana',
+        'Ville',
+        'Niina',
+        'Marjo',
+        'Napoleon',
+        'Maria',
+        'Kari',
+        'Sari',
+        'Antti',
+        'Janne',
+        'Teppo',
+        'Matti',
+        'Joonas',
+      ],
+      frames: [{
+        subframes: JSON.parse(JSON.stringify(TEST_FRAMES)),
+        notes: ''
+      }, { subframes: JSON.parse(JSON.stringify(TEST_FRAMES)), notes: '' },
+        {
+          subframes: JSON.parse(JSON.stringify(TEST_FRAMES)),
+          notes: ''
+        }, { subframes: JSON.parse(JSON.stringify(TEST_FRAMES)), notes: '' }, {
+          subframes: JSON.parse(JSON.stringify(TEST_FRAMES)),
+          notes: ''
+        }, { subframes: JSON.parse(JSON.stringify(TEST_FRAMES)), notes: '' },
+        {
+          subframes: JSON.parse(JSON.stringify(TEST_FRAMES)),
+          notes: ''
+        }, { subframes: JSON.parse(JSON.stringify(TEST_FRAMES)), notes: '' }],
+      carpet: {
+        color: '#5151b8',
+        height: 8,
+        width: 8,
+        horizontalSegments: 8,
+        verticalSegments: 4
+      }
+    }];
+
+  constructor() {
+  }
+
+  getChoreographies(): Observable<Choreography[]> {
+    return timer(1000).pipe(mapTo(this.choreographies));
+  }
+
+  getChoreographiesById(id: number): Observable<Choreography> {
+    return timer(500).pipe(mapTo(this.choreographies[id - 1]));
```

Comment:
**issue:** your accessing nth item instead of the one with the specific id (use `.find(choreography => choreography.id === id)`)